src/sagas.js

```diff
@@ -700,6 +697,11 @@ function* schedulerPerMinute (action) {
   }
 }
 
+function* delayedRemoveActiveRequest ({ payload: requestId }) {
```

Comment:
I think this action doesn't give any sense, because chance, that next request will be in 1 ms after `removeActiveRequest`, the same as if next request will be in 501 ms.

Reply:
It's there to prevent most of the flickering (practically all of it). Sure there can still occur blinks. Any ideas how to improve that?

Reply:
Maybe I miss something, but this delay improves blinking significantly.
The root cause for it is mostly in "batch" load composing of multiple calls, so executed tightly together.

My first guess was one or two seconds, but let's see - 500ms might be good enough